app/src/debug/java/com/artemzin/qualitymatters/ui/fragments/DeveloperSettingsFragment.java

```diff
@@ -105,6 +108,12 @@ void onHttpLoggingLevelChanged(int position) {
         presenter.changeHttpLoggingLevel(((HttpLoggingLevel) httpLoggingLevelSpinner.getItemAtPosition(position)).loggingLevel);
     }
 
+    @OnClick(R.id.developer_settings_restart_app)
+    void onRestartAppClick() {
+        final FragmentActivity activity = getActivity();
+        ProcessPhoenix.triggerRebirth(activity, new Intent(activity, activity.getClass()));
```

Comment:
Well, yeah, though just as note: we can wrap ProcessPhoenix into separate class and invoke it from the presenter, but I'm ok with current solution